unstructured/partition/csv.py

```diff
@@ -29,6 +30,7 @@ def partition_csv(
     metadata_filename: Optional[str] = None,
     metadata_last_modified: Optional[str] = None,
     include_metadata: bool = True,
+    languages: List[str] = ["auto"],
```

Comment:
Does the csv partitioner not have the `detect_language_per_element` param because it only results in one table element? Or is there another reason `detect_language_per_element` isn't part of every partitioner?

Reply:
Yah, I left it off because the partitioner returns single TableElements. In working on this, though, I did realize there are separate elements for multiple pages in an xlsx doc, though. So I could create a file with different languages on different sheets, but that seemed like an unnecessary/unlikely test/use case

Reply:
Ok, that makes sense - yeah the excel diff languages across sheets is fine to skip for now :) Could be worth adding a comment about why the `detect_language_per_element` param pattern didn't apply here, or apply it anyway for consistency with a comment that it doesn't get used since only a single element is returned.

Reply:
Sounds good. I have one in the tests, but it would be better to have a comment in the actual partitioner